aidants_connect_web/models.py

```diff
@@ -13,7 +13,7 @@ def default_expiration_date():
     return now + timedelta(minutes=CONNECTION_EXPIRATION_TIME)
 
 
-class User(AbstractUser):
+class Aidant(AbstractUser):
     profession = models.TextField(blank=False)
     organisme = models.TextField(blank=False)
     ville = models.TextField(blank=False)
```

Comment:
If you add : 
```py
    class Meta:
        verbose_name = "aidant"
```
https://docs.djangoproject.com/en/dev/ref/models/options/#verbose-name
It will be displayed as `Aidant`/`Aidants` in the admin page instead of `Utilisateur`/`Utilisateurs`